app/views/liquids/_user_subscription.html.erb

```diff
@@ -71,9 +71,8 @@
           </header>
           <div class="crayons-modal__box__body">
             <p id="confirmation-text" class="fs-base mb-4 mt-0">
-              You'll share your email address
-              <span class="ltag__user-subscription-tag__subscriber-email"></span>
-              with
+              You'll share your email address<span class="ltag__user-subscription-tag__subscriber-email"></span>,
+              username, name, and DEV profile URL with
```

Comment:
Being especially clear about what the user/subscriber will be sharing. Check out the screenshot in #9052 to see how this info is used.